### ISSUE TITLE
Support new languages without code

### ISSUE THREAD

**Issue by colemarkham** [dschaefer/antlng#3]
Since we're talking about dreams, can this be extensible without requiring custom development? I don't know enough about Antlr to know if this is even possible, but one thing I've always wanted is to provide a grammar and get an editor with syntax validation and highlighting at the very least. Ideally this would be done at runtime, but if code generation is required an acceptable compromise would be to run the grammar through a build tool that produces an installable plugin.

**Comment by scharf** [comment]
This is what [LiClipse](http://www.liclipse.com) does. It supports TextMate syntax highlighting bundles.

**Comment by dschaefer** [comment]
We really need this functionality to be in the Eclipse platform, not someone's commercial add-on. As much as I appreciate the ecosystem, there are too many free and open systems that do that and Eclipse is falling behind.

**Comment by dschaefer** [comment]
I checked out the TextMate Manual. If LiClipse doesn't make it to Eclipse, it actually doesn't look that hard to be able to rebuild it.

https://manual.macromates.com/en/language_grammars

**Comment by scharf** [comment]
Yes I have done something similar for the wind river IDEs. We had an editor that would take a file with declarations to do the syntax highlight. The niche thing is , once you have the binding with the editor done, adding a new language is almost trivial.
